src/app/sitemap.xml/route.ts

```
@@ -0,0 +1,51 @@
+    }`,
+    params: { baseUrl: `${baseUrl}/` },
+  });
+
+  const all = Object.values(data).flat();
+
+  // Build XML
+  let xml = `<?xml version="1.0" encoding="UTF-8"?>\n`;
+  xml += `<?xml-stylesheet type="text/xsl" href="/sitemap.xsl"?>\n`;
+  xml += `<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">\n`;
+  for (const entry of all) {
+    xml += `  <url>\n`;
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)